LeaderboardBackend/Controllers/Requests/Leaderboards/Create.cs

```diff
@@ -1,9 +1,7 @@
-namespace LeaderboardBackend.Controllers.Requests
-{
-	public class CreateLeaderboardRequest
-	{
-		public string Name { get; set; } = null!;
+namespace LeaderboardBackend.Controllers.Requests;
 
-		public string Slug { get; set; } = null!;
-	}
+public class CreateLeaderboardRequest
```

Comment:
Record types could actually work really well for these request models. Up to you if you'd like to make that change as part of this PR.

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d (`Create.cs` [`R3`](https://github.com/leaderboardsgg/leaderboard-backend-poc/commit/76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d#diff-d8fbeab83ddcb8ed78d9a8c3bf342cdbd1dee612296dd03ee1f2624bdb5b97d4R3)) and others.